getMedia.js

```diff
@@ -107,12 +121,12 @@ const resolvers = {
       if (err) return next(providers, mediaURL, options, firstErr || err, callback)
 
       metascraper.scrapeHtml(body).then((result) => {
+        console.log('result: ' + JSON.stringify(result, null, 2))
```

Comment:
this contains potentially sensitive information about the page and should not be logged by default

Reply:
oops! debug thing not removed. fixed.